### ISSUE TITLE
Remove the `aria-expanded` attribute from the menu 

### ISSUE THREAD

**Issue by afercia** [WordPress/twentynineteen#270]
The `aria-expanded` is a _state_, not a property, and it's meant to communicate a change in the state:

https://www.w3.org/TR/wai-aria-1.1/#aria-expanded
> Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed.

As far as I see, in Twenty Nineteen `aria-expanded` is never updated and it's always `false`. As such, it gives an incorrect information to assistive technology users. For example, when a menu tree is open, it's still announced as "collapsed":

![screen shot 2018-10-24 at 16 37 19](https://user-images.githubusercontent.com/1682452/47439631-02239300-d7ad-11e8-92f6-fadf78c487a6.png)

However, I'd suggest to remove `aria-expanded` entirely as it is better suited for UI controls that require _user activation_. As in:
- users activate an UI control
- `aria-expanded` communicates a change in the state (expanded, or collapsed)
- users are informed something expanded or collapsed in the page 
- they can now explore around for the new content or be aware some previously displayed content has been removed

Instead, the Twenty Nineteen menu doesn't require explicit user interaction to expand or collapse the sub-trees. It is sufficient to navigate normally pressing Tab or hover with the mouse to get the new content.

**Comment by peterwilsoncc** [comment]
@afercia In #280 the illustration for the mobile menus shows a user action is required to open menus and submenus. Will `aria-expanded` be required for that layout?

**Comment by nielslange** [comment]
@afercia In addition to `aria-expanded` I also removed `aria-haspopup` as both elements came together and none of them changed its state.

**Comment by afercia** [comment]
@peterwilsoncc thanks, yep if in the mobile view the menu requires activation then `aria-expanded` should communicate the state change and should be used only in the mobile view. 😱 

@nielslange thanks, I'm undecided about [aria-haspopup](https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup) as it's not a state, it's a property to indicate if there's a popup or not. However, to use it properly, a few other things should be changed and I'd rather recommend to keep things simple.

For the records, the [previous wording of the ARIA spec](https://www.w3.org/2016/04/28-aria-minutes.html#item03) stated that it should be used when "activation renders conditional content". The current wording is different and doesn't fully clarify this point (at least not to me), it just says "Indicates the availability and type of **interactive** popup element".

**Comment by nielslange** [comment]
> However, to use it properly, a few other things should be changed and I'd rather recommend to keep things simple.
Thus, remove it for now and potentially introduce it again with an adjusted implementation in v1.1. 😉
